source/content/guides/integrated-composer/05-ic-upstreams.md

```diff
@@ -29,47 +29,31 @@ Upstream refers to the source code that is hosted in the Pantheon code repositor
 
 ### How to Add Dependencies to Your Upstream
 
-1. [Clone the Git repository](/guides/git/git-config#clone-your-site-codebase) from the Site Dashboard.
+1. Clone the Git repository for your upstream.
 
-1. Change into the Upstream's configuration directory:
-
-   - Drupal:
-
-    ```bash{promptUser: user}
-    cd upstream-configuration
-    ```
-
-   - WordPress:
+1. Run the `composer upstream-require` command for each dependency:
 
     ```bash{promptUser: user}
-    cd upstream-config 
+    composer upstream-require drupal/pkg-name [--no-update]
     ```
 
-1. Run the `composer require` command for each dependency:
+     - `--no-update` tells Composer to disable automatic updates of the dependency. This makes Composer faster when adding dependencies to the Upstream as shown here. This is useful if you don't want to pin versions for your dependencies. Do not include this option if you want to pin specific versions for your dependencies.
```

Comment:
It's not necessary to specify `--no-update` here; the script will automatically include `--no-update` if you're not pinning versions, and it will automatically put in `--no-install` if you are.

Use `composer update-upstream-dependencies` to start pinning changes.